code/modules/cm_marines/dropship_equipment.dm

```diff
@@ -693,9 +694,16 @@
 
 	// clamp back to maximum inaccuracy
 	ammo_accuracy_range = min(ammo_accuracy_range, ammo_max_inaccuracy)
-
 	var/list/possible_turfs = RANGE_TURFS(ammo_accuracy_range, target_turf)
 	var/turf/impact = pick(possible_turfs)
+
+	if(ammo_travelling_time && istype(SA, /obj/structure/ship_ammo/rocket/custom_missile))
+		playsound(impact, ammo_warn_sound, ammo_warn_sound_volume, 1,15)
+		var/total_seconds = max(round(ammo_travelling_time/10),1)
+		for(var/i = 0 to total_seconds)
```

Comment:
for(var/i in 0 to some_value) is 99% the same thing but it looks more slick no? :clueless:

Reply:
chat is this real

Reply:
The = form should generally be this: https://www.byond.com/docs/ref/#/proc/for/loop
Otherwise the numeric form at the bottom here: https://www.byond.com/docs/ref/#/proc/for/list

Reply:
done